types/i/i-tests.ts

```diff
@@ -0,0 +1,147 @@
+import i = require('i');
+import "i/globals";
```

Comment:
Just to make this more clear for consumers, you should use a `/// <reference types="i/globals" />` instead. This import would fail at runtime, after all.

Reply:
Should I add this reference in test file or directly in index.d.ts? Because if I add it in index file its definitions are available always, doesn't matter if I set true to add them